### ISSUE TITLE
Can support dynamic charts? Make the cell black。

### ISSUE THREAD

**Issue by nathanQiu1** [ClosedXML/ClosedXML#778]
## Read and complete the full issue template

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML**

0.93.0

**What is the current behavior?**

The dynamic chart disappeared，Cell color changed to black。

**What is the expected behavior or new feature?**

Cell color is  normal， dynamic chart is normal。

## Reproducibility
**This is an important section. Read it carefully. Failure to do so will cause a 'RTFM' comment.**

Without a code sample, it is unlikely that your issue will get attention. Don't be lazy. Do the effort and assist the developers to reproduce your problem. Code samples should be [minimal complete and verifiable](https://stackoverflow.com/help/mcve). Sample spreadsheets should be attached whenever applicable. Remove sensitive information.

**Code to reproduce problem:**
```c#
public void Main()
{
using (XLWorkbook workbook = new XLWorkbook(@"FAI.xlsm"))
   {
       workbook.Save();
   }
}
```
- [x] I attached a sample spreadsheet.  (You can drag files on to this issue)
[dynamiccharts.xlsx](https://github.com/ClosedXML/ClosedXML/files/1903078/dynamiccharts.xlsx)

**Comment by nathanQiu1** [comment]
Updated the latest code to support charts. the black part still exists.

**Comment by Pankraty** [comment]
@nathanQiu1 please check the attached file. From my perspective, it looks OK.

[778_result.xlsx](https://github.com/ClosedXML/ClosedXML/files/2019816/778_result.xlsx)

**Comment by nathanQiu1** [comment]
@Pankraty   thank you for your reply!  After the file passes closedxml, there will be a black area.
![image](https://user-images.githubusercontent.com/19723900/40285703-d4dc4f0a-5cd1-11e8-816d-3cc8e1e501dc.png)

**Comment by Pankraty** [comment]
The file I attached came from ClosedXML with fixes from #862. If there are no other issues with this file then this bug will be closed when 862 is merged.

**Comment by nathanQiu1** [comment]
OK, thank you very much!

**Comment by igitur** [comment]
Fixed in #862
